docs/api/native-image.md

```diff
@@ -137,15 +137,17 @@ let image = nativeImage.createFromPath('/Users/somebody/images/icon.png')
 console.log(image)
 ```
 
-### `nativeImage.createFromBuffer(buffer[, scaleFactor])`
+### `nativeImage.createFromBuffer(buffer[width, height, scaleFactor])`
 
 * `buffer` [Buffer][buffer]
+* `width` Integer (optional)
```

Comment:
What do you think about making this new version `createFromBuffer(buffer, options)` and width, height and scaleFactor can be specified in there?

Then it is easier to add new keys/options in the future and we can just try parsing the second argument as an object and if it fails we fall back to a scale factor second argument with a default width/height.